Vagrantfile

```diff
@@ -0,0 +1,51 @@
+Vagrant.configure("2") do |config|
+  config.ssh.forward_agent = true
+
+  config.vm.define "ubuntu" do |node|
+    node.vm.box = "ubuntu/bionic64"
```

Comment:
Lets use the bento box?

Reply:
![image](https://user-images.githubusercontent.com/575693/44228858-98a35a80-a153-11e8-9ea5-2d64d5eb3784.png)

Reply:
Rassafrappit! I don't know why I keep getting tripped up on bringing in bentos.

Yes, of course.